May_2020_Meeting

```diff
@@ -0,0 +1,28 @@
+Date: 05/??/2020
+Time: ??:??
+Coordinator: Ran
+Summury by: ??
```

Comment:
Summary

Reply:
Here it is written well:
https://github.com/wg21il/Protocols/pull/5/files#diff-7a9eb2303f561f2d2a6ed17460c4b115R15
(also in the commit message it is correct)